dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/OpenTelemetryProducer.java

```diff
@@ -95,6 +99,11 @@ public OpenTelemetry getOpenTelemetry() {
         //By default, MicroProfile Telemetry tracing is off.
         //The absence of an installed SDK is a “no-op” API
         //Operations on a Tracer, or on Spans have no side effects and do nothing
+        ComponentMetaData cData = ComponentMetaDataAccessorImpl.getComponentMetaDataAccessor().getComponentMetaData();
+        String applicationName = cData.getJ2EEName().getApplication();
+        String msg = Tr.formatMessage(tc, "CWMOT5100.tracing.is.disabled", applicationName);
+        Tr.info(tc, msg);
+        
         return OpenTelemetry.noop();
```

Comment:
What does the application name look like when you get it like this?

Can you copy the output line from a test run so we can check it looks right?

Reply:
`[21/02/2023, 14:44:09:445 GMT] 0000005a ty.microprofile.telemetry.internal.cdi.OpenTelemetryProducer I CWMOT5100I: The MicroProfile Telemetry Tracing feature is enabled but has not been configured to create traces for the TelemetryDisabledTracingApp application.`

Reply:
Actually I think we can see it well enough in your test:

```
assertNotNull(server.waitForStringInLogUsingMark("CWMOT5100I: The MicroProfile Telemetry Tracing feature is enabled but has not been configured to create traces for the " + APP_NAME + " application."));
```